src/backend_module.erl

```diff
@@ -102,27 +102,29 @@ fun_header(F, Args) ->
     [atom_to_list(F), "(", Args, ")"].
 
 time_metric(Module, Op) -> [backends, Module, Op].
-count_metric(Module, Op) -> [backends, Module, Op, count].
+calls_metric(Module, Op) ->
+    Suffix = list_to_atom(atom_to_list(Op) ++ "_calls"),
```

Comment:
It's not really necessary to create suffix in such a way. I'd recommend changing name of the metrics to `[backends, Module, calls, Op]`.